apps/labs/posts/lets-make-pygments-accessible.md

```diff
@@ -0,0 +1,70 @@
+---
+title: '"Let's make Pygments accessible!"'
+author: stephannie-jimenez
+published: November 15, 2022
+description: 'accessible-pygments hosts curated WCAG-compliant themes for all your syntax highlighting needs.'
+category: [Access-centered]
+featuredImage:
+  src: /posts/lets-make-pygments-accessible/featured_themes.png
+  alt: 'An animation of different accessible syntax highlighting themes being applied to the same example code.'
```

Comment:
```suggestion
  alt: 'A display of different accessible syntax highlighting themes being applied to the same example code.'
```

since it is not an animation anymore